doc/usage/integration.rst

```diff
@@ -21,28 +21,28 @@ recommended for all applications that support Geant4 11.0 or higher.
 1. Find and link in your CMake project:
```

Comment:
Not this line, but for the Usage overview right above it (that github doesn't let me comment on):

Since the whole section focuses on the tracking manager integration, it may be useful to point to the documentation to the user action case. Something like

> for all applications that support Geant4 11.0 or higher. For older versions, refer to the user action integration documentation and example in `offload-template`

Reply:
The offload template uses the tracking manager, but I'll add a reference to the manual mentioning the other support.

Reply:
Oh right, we also moved to the TMI during its PR.